Makefile

```diff
@@ -4,15 +4,21 @@
 # SPDX-License-Identifier: MIT
 
 CC	= cc
-INC	= -I/global/common/software/nersc9/nccl/2.19.4/include
-CFLAGS	= -std=c++11 -O2 -target-accel=nvidia80 --cuda-gpu-arch=sm_80 -DUSE_CUDA -DUSE_NCCL
-LDFLAGS	= -L/global/common/software/nersc9/nccl/2.19.4/lib -lnccl
 
+# perlmutter flags
+# INC	= -I/global/common/software/nersc9/nccl/2.19.4/include
+# CFLAGS	= -std=c++11 -O2 -target-accel=nvidia80 --cuda-gpu-arch=sm_80 -DUSE_CUDA -DUSE_NCCL
+# LDFLAGS	= -L/global/common/software/nersc9/nccl/2.19.4/lib -lnccl
+
+# frontier flags
+INC = -L${ROCM_PATH}/lib -lamdhip64
```

Comment:
1. Both of these should probably go to LDFLAGS.
2. do we not want to add something like -I${ROCM_PATH}/include like we do for perlmutter